src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2006.UrisShouldBeReachable_Valid.sarif

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "version": "1.0",
+          "downloadUri": "https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html",
```

Comment:
>https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html [](start = 26, length = 64)

Let's make this [https://github.com/microsoft/sarif-sdk](https://github.com/microsoft/sarif-sdk). I don't want to ping OASIS.

Reply:
I do understand that the validator always uses the OASIS copy of the spec for the `helpUri` so there's nothing you can do about that.

---
In reply to: [446546315](https://github.com/microsoft/sarif-sdk/pull/1942#discussion_r446546315) [](ancestors = 446546315)